app/services/authentication.py

```diff
@@ -13,7 +17,7 @@ def decode_token(token, expiration=3600):
     try:
         email = serializer.loads(token, salt="email-confirm-salt", max_age=expiration)
         return email
-    except Exception as e:
+    except Exception:
```

Comment:
Why are we capturing `Exception` here.  Is there nothing higher up we can catch?